### DISCUSSION TITLE
Read this blog from 2021 - very inspiring

### DISCUSSION THREAD

**Discussion by gganapat** [observablehq/framework#1079] [General]
Re-read [this]([url](https://observablehq.com/@mbostock/10-years-of-open-source-visualization)) blog from 2021! 

If you haven’t read it, it’s worth your time reading it. Besides being a verified genius, Mike is a true inspiration. God bless him!!

gani-

**Comment by gganapat** [comment]
Oops link - https://observablehq.com/@mbostock/10-years-of-open-source-visualization

**Comment by declann** [comment]
I took your advice - thanks for sharing!,

> Besides being a verified genius, Mike is a true inspiration. God bless him!!

Too correct!
